scylla/tests/integration/query_result.rs

```diff
@@ -0,0 +1,223 @@
+use futures::TryStreamExt;
+use scylla::{
+    batch::{Batch, BatchType},
+    client::session::Session,
+    query::Query,
+};
+use scylla_cql::frame::request::query::{PagingState, PagingStateResponse};
+
+use crate::utils::{create_new_session_builder, setup_tracing, unique_keyspace_name, PerformDDL};
+
+const PAGE_SIZE: i32 = 100;
+const ROWS_PER_PARTITION: i32 = 1000;
+const PARTITION_KEY1: &str = "part";
+const PARTITION_KEY2: &str = "part2";
+
+/// Initialize a cluster with a table and insert data into two partitions.
+/// Returns a session and the keyspace name.
+///
+/// #Example
+/// ```
+/// let (session, ks) = initialize_cluster().await;
+/// ```
+async fn initialize_cluster_two_partitions() -> (Session, String) {
```

Comment:
:wrench: naming mismatch.

Reply:
```suggestion
/// # Example
/// ```rust
/// let (session, ks) = initialize_cluster_two_partitions().await;
/// ```
async fn initialize_cluster_two_partitions() -> (Session, String) {
```